app/src/tests/mocks/mockRequests.ts

```diff
@@ -0,0 +1,8 @@
+import { FilesRequestBody } from 'src/shared/interfaces';
+
+export const processFilesMockRequest: FilesRequestBody = {
```

Comment:
Suggestion: use the newly defined interface here (and probably remove unused stuff)

Reply:
The request and response body objects are not the same. I think it would be best to keep both and perhaps rename FilesRequestBody to SpecMathRequest for clarity.